README.md

```diff
@@ -23,6 +21,18 @@ Screenshots
 Installing Piper
 ================
 
+Most popular distributions package Piper and it is available through the
+packaging system (apt, dnf, yum, pacman, ...). This is the preferred way of
+installing Piper.
+
+Piper is also available as a
+[Flatpak]([200~https://flathub.org/apps/details/org.freedesktop.Piper).
```

Comment:
Just noticed that this link isn't showing in the rendered README.. sorry

Reply:
doh, should've checked. fixed now